test/hotspot/jtreg/runtime/ClassUnload/SuperDependsTest.java

```diff
@@ -75,9 +77,8 @@ public void test() throws Throwable {
     public static void main(String args[]) throws Throwable {
         SuperDependsTest d = new SuperDependsTest();
         d.test();
-        ClassUnloadCommon.triggerUnloading();  // should not unload anything
         System.out.println("Should unload MyTest and p2.c2 just now");
```

Comment:
With the removal of line 78 the println is no longer relevant/correct/useful.

Reply:
removed.